src/helpers/i18n.js

```diff
@@ -27,9 +27,11 @@ export function formatAmount(intl, amount) {
 }
 
 export function formatDateFromISO(mm, intl, date) {
-  let isSecondaryCalendar = JSON.parse(localStorage.getItem('isSecondaryCalendarEnabled'));
+  const isSecondaryCalendar = JSON.parse(localStorage.getItem("isSecondaryCalendarEnabled"));
   if (isSecondaryCalendar) {
-    return neFormateDateFromISO(date);
+    const secondCalendarFormatting = mm.getConf("fe-core", "secondCalendarFormatting", "DD-MM-YYYY");
+    const secondCalendarFormattingLang = mm.getConf("fe-core", "secondCalendarFormattingLang", "en");
```

Comment:
"secondCalendarFormattingLang" could be moved to consts:
`
const DEFAULT_CALENDAR_LANG = "secondCalendarFormattingLang";
`